documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1220,11 +1220,12 @@ In the `source` object, `ts_ms` indicates the time that the change was made in t
 In case a single `TRUNCATE` statement applies to multiple tables,
 one _truncate_ change event record for each truncated table will be emitted.
 
-Note that since _truncate_ events represent a change made to an entire table and don't have a message key,
-unless you're working with topics with a single partition,
-there are no ordering guarantees for the change events pertaining to a table (_create_, _update_, etc.) and _truncate_ events for that table.
-For instance a consumer may receive an _update_ event only after a _truncate_ event for that table,
-when those events are read from different partitions.
+[NOTE]
+====
+A _truncate_ event represents a change that is made to an entire table and it has no message key.
+As a result, for topics with multiple partitions, there are no ordering guarantees for the change events (_create_, _update_, and so forth), or _truncate_ events that pertain to a table.
+For example, when a consumer reads events for a table from multiple partitions, it might receive an _update_ event for a table only after it receives a _truncate_ event that deletes all of the data in the table.
+====
```

Comment:
I also touched this text in #6016, not considering that in that PR I'd want to apply a set of changes that were consistent with other connectors. Where there's a collision, the changes introduced in 6016 should win out.